### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.servicefabric.models.ValidationFailedChaosEvent
  fullName: azure.servicefabric.models.ValidationFailedChaosEvent
  inheritance:
  - inheritance:
    - inheritance:
      - type: builtins.object
      type: msrest.serialization.Model
    type: azure.servicefabric.models._models_py3.ChaosEvent
  langs:
  - python
  module: azure.servicefabric.models
  name: ValidationFailedChaosEvent
  summary: 'Chaos event corresponding to a failure during validation.


    All required parameters must be populated in order to send to Azure.'
  syntax:
    content: 'ValidationFailedChaosEvent(*, time_stamp_utc, reason: str = None, **kwargs)
      -> None'
    parameters:
    - description: 'Required. The UTC timestamp when this Chaos event

        was generated.'
      id: time_stamp_utc
      type:
      - datetime
    - description: Required. Constant filled by server.
      id: kind
      type:
      - str
    - description: 'Describes why the ValidationFailedChaosEvent was generated.

        This may happen because more than MaxPercentUnhealthyNodes are unhealthy

        for more than MaxClusterStabilizationTimeout. This reason will be in the

        Reason property of the ValidationFailedChaosEvent as a string.'
      id: reason
      type:
      - str
  type: class
  uid: azure.servicefabric.models.ValidationFailedChaosEvent
references: []
